### ISSUE TITLE
Can't unlock archives

### ISSUE THREAD

**Issue by smxsm** [buttercup-pw/buttercup-browser-extension#54]
Hi, first of all, thanks for Buttercup, overall looking very good so far :)
I have a problem with the Chrome extension though, if an archive is locked, nothing happens if I click the Buttercup Icon in the browser bar, so I'm not able to unlock my archive.
Somehow it worked after a while yesterday, but today I've clicked on the "lock" icon for testing and now I can't unlock it again since the "flowout" with the archive doesn't showup on click :(
Any idea what might be the problem?
![image](https://cloud.githubusercontent.com/assets/1737182/24541109/5f44d708-15f6-11e7-9feb-51d510eb2465.png)

**Comment by perry-mitchell** [comment]
Sounds like a very weird issue, or one that I'm not understanding. Are you saying that you're not even seeing the popup appear when you click the button?

![image](https://cloud.githubusercontent.com/assets/3869469/24554433/2e018ae8-1636-11e7-9ef4-a49585f2b415.png)

It is possible that there's some error occurring in the popup which prevents it from rendering.. but to debug we'd need to be able to get an inspector open on the popup:

![image](https://cloud.githubusercontent.com/assets/3869469/24554470/5ca3979c-1636-11e7-8245-6781a5697fdf.png)

Is there any way you could open the console in the popup to see if there are any errors?

**Comment by smxsm** [comment]
Yep, no popup at all, IF the archive is locked... if it is unlocked, I get the popup... 
There are no errors in the console, but I can't inspect it, since there is nothing showing up :(
I am on MAC OS, Chrome Version is 57.0.2987.133 (64-bit) btw., using a single WebDAV archive.

Strange thing is, if I lock it by clicking on it, there seems to be a Javascript alert, but it disappears within an instant ... and after that, there is no popup anymore if I click to re-authorize the archive. I then have to re-activate the plugin, then I can open the popup and enter the master password again.

**Comment by perry-mitchell** [comment]
Ok.. Dang. Ok, what about inspecting the background page for any weirdness:

![image](https://cloud.githubusercontent.com/assets/3869469/24555085/84276aee-1638-11e7-9052-74e551034c9a.png)

In the extensions page you can just click on the "background page" link to see the console for the background script. Is there anything there?

Furthermore: Did you install the extension from the store, or are you running it from a master clone or something else? It'd be great if you could debug the popup if possible..

**Comment by smxsm** [comment]
Hm, nothing in the background log either :( I have also disabled all other plugins, but the error still occurs ... weird. I am using the official Chrome Store version.

**Comment by smxsm** [comment]
Interestingly, I tried to lock the WebDAV archive on Windows Chrome, and I've had an error message, too... but there it stayed, to I was able to make a screenshot:
![buttercup_error_win](https://cloud.githubusercontent.com/assets/1737182/24623232/09f73b2e-18a8-11e7-8828-338d30ff60a4.png)

I don't know if that is related to the MAC problem, though... just wanted to let you know :)
Not sure why it reports a timeout though, the error came up instantly ...

**Comment by perry-mitchell** [comment]
@smxsm What webdav server are you using? I wonder if there's some weird issue tied to the rendering of the popup..

Can you access [this page](chrome-extension://heflipieckodmcppbnembejjmabajjjj/setup.html#/) ?

**Comment by smxsm** [comment]
It's our own server... the WebDAV connection is working and I can also access the setup page and see the archive listed. It's only a problem if I try to lock the archive.

**Comment by perry-mitchell** [comment]
Hi @smxsm - At this stage I'm not sure that there's more I can ask you that would help debug this. I have 2 working webdav connections that I frequently test with and I haven't encountered an issue like this yet. I would ask you to debug it further in a development setup, but if you're not interested or don't have the time I fully understand. Aside from that I don't think I can currently help, sorry.
